### ISSUE TITLE
Extra whitespace on cards in new gallery

### ISSUE THREAD

**Issue by brian-rose** [ProjectPythia/projectpythia.github.io#118]
The [new links gallery](https://projectpythia.org/gallery.html) is displaying cards with seemingly random heights. Some cards have a lot of extra whitespace.

Screenshot of the page rendered on Safari:
![Screen Shot 2021-06-22 at 12 16 38 PM](https://user-images.githubusercontent.com/9097307/122963199-0f292380-d354-11eb-8e31-fe97e3713157.png)

**Comment by brian-rose** [comment]
... but the same page rendered on Firefox looks better:

<img width="1511" alt="Screen Shot 2021-06-22 at 12 20 45 PM" src="https://user-images.githubusercontent.com/9097307/122963499-51526500-d354-11eb-9d5d-9db72080c997.png">

So maybe this is a Safari-specific bug?

**Comment by kmpaul** [comment]
Yes.  It looks like the Firefox display on Chrome, as well.  I'd say this is a Safari problem.  I'll see if I can diagnose it.

**Comment by kmpaul** [comment]
On my laptop's Safari browser, it looks like this:

<img width="1266" alt="Screen Shot 2021-06-22 at 10 45 12 AM" src="https://user-images.githubusercontent.com/11411331/122966805-ffefa900-d346-11eb-9286-e1fb8742c1cf.png">

What version of Safari are you using?  I am using Version 14.1.1 (15611.2.7.1.6, 15611).

**Comment by brian-rose** [comment]
Safari 14.0.3 (15610.4.3.1.7, 15610) on macOS Catalina 10.15.17

**Comment by kmpaul** [comment]
And my laptop is Catalina 10.15.7.

**Comment by kmpaul** [comment]
Have you done a force reload (Command+R) on the page?

**Comment by brian-rose** [comment]
Yes, no change.

I can try installing the latest Safari update.

**Comment by kmpaul** [comment]
You could, but I don't think that's the solution we want.  And the problem I have is that I cannot debug the issue.  😦

**Comment by brian-rose** [comment]
Just checked my other machine:
Safari 14.1.1 (15611.2.7.1.6, 15611) on MacOS Catalina 10.15.17
Here's what it looks like:
<img width="1932" alt="Screen Shot 2021-06-22 at 3 24 02 PM" src="https://user-images.githubusercontent.com/9097307/122986886-fcbbe380-d36d-11eb-9f5b-03d68a10b26d.png">

So it seems the problem is limited to an outdated version of Safari!

Maybe best to ignore it if there's no obvious fix.

**Comment by kmpaul** [comment]
But it doesn't seem *that* outdated.  😦 

Does anyone else see this with their default browsers?

**Comment by brian-rose** [comment]
Hmm. I know basically nothing about CSS, but I see in the [Safari release notes](https://developer.apple.com/documentation/safari-release-notes/safari-14_1-release-notes) for version 14.1:
> Added support for CSS individual transform properties: translate, rotate, and scale.

and I see what looks like a `scale` transform in our code here:
https://github.com/ProjectPythia/projectpythia.github.io/blob/0388706c249e1270e5b36b1c77e78f70eae19357/sphinx_pythia_theme/static/css/pythia.css#L2595

Could that have something to do with it?

**Comment by ktyle** [comment]
The gallery page looks good on all of the browsers I have on my Windows 10 box: Chrome, Edge, Firefox, and Opera. 
Also looks good on Firefox, using a laptop running Fedora 34.

**Comment by kmpaul** [comment]
@brian-rose:  I'm not entirely sure, but I don't think that those CSS classes are being used on the page, so I don't think that is it.  But I could be wrong.  I'm not a web developer, either.  ☹️

**Comment by ktyle** [comment]
Yes, I vote for closing. Perhaps we could put create a FAQ where this could be one of the potential issues a user might encounter?
